css/_chat.scss

```diff
@@ -1,6 +1,6 @@
 #sideToolbarContainer {
     background-color: $newToolbarBackgroundColor;
-    display: flex;
```

Comment:
Why did you decide to remove flex from here?

Reply:
This was part of the changes that tried to focus on CSS only and I removed it because it's not doing anything right now. I could make it do something by cleaning up the DOM. So maybe I'll make that change in the other PR.

Reply:
I changed the base PR. I'll resolve whatever conflicts happen after that side is merged.